packages/expo-font/src/ExpoFontLoader.ts

```diff
@@ -1,2 +1,12 @@
 import { requireNativeModule } from 'expo-modules-core';
-export default requireNativeModule('ExpoFontLoader');
+
+const m =
+  typeof window === 'undefined'
+    ? // React server mock
+      {
+        getLoadedFonts() {
+          return [];
+        },
```

Comment:
the native module also exposes `loadAsync` function. Even if it never gets called on server, I'd implement it here and have it throw some error?

Reply:
what's the purpose of having it throw an error?

Reply:
Getting an error that tells that what I'm doing is not intended to work and I'll understand that right away, as opposed to getting "undefined is not a function".